### PR TITLE
task16

### PR DESCRIPTION
**Pull request by SafaaMagdyAwad**
hi eng-Nourhan , i created a job ,  could you please take a look at the code and tell me your comment about it ?

### CODE REVIEW

**Comment by Nora-Lootfy** [comment]
> hi eng-Nourhan , i created a job , could you please take a look at the code and tell me your comment about it ?

for sure😊

File: app/Console/Commands/BackupDatabase.php [hunk 1]
```diff
@@ -25,11 +25,11 @@ class BackupDatabase extends Command
      */
     public function handle()
     {
-        $databaseName = env('DB_DATABASE');
-        $username = env('DB_USERNAME');
-        $password = env('DB_PASSWORD');
-        $host = env('DB_HOST');
-        $port = env('DB_PORT', '3306');
+        $databaseName = config('database.connections.mysql.database');
```
Comment (Nora-Lootfy) [inline app/Console/Commands/BackupDatabase.php:28]: nice refactoring😊, thanks for considering

File: app/Http/Controllers/BaseController.php [hunk 2]
```diff
@@ -30,8 +32,9 @@ class BaseController extends Controller
 
     public function index(): View
     {
+        $user=Auth::user();
         $model = new $this->model;
-
+        WelcomeEmailJob::dispatch($user);
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/BaseController.php:37]: a little extra  code but a bit faster application

**Review by Nora-Lootfy — APPROVED** [review]
I truly appreciate the time, energy, and passion you bring to everything you do.

Thanks ya @SafaaMagdyAwad , hoping the best for U😊

File: app/Http/Controllers/BaseController.php (hunk 2 above)
Comment (SafaaMagdyAwad) [reply app/Http/Controllers/BaseController.php:37]: Is there a  A simpler way

Comment (Nora-Lootfy) [reply app/Http/Controllers/BaseController.php:37]: no, your code looks very neat  and also you make it faster

**Pull request merged**